### ISSUE TITLE
Move code to main repo?

### ISSUE THREAD

**Issue by molant** [webhintio/webhint-ado-extension#15]
It could benefit from the release script, linters, etc.

@webhintio/core thoughts?

**Comment by antross** [comment]
Makes sense to me. Would we put it under `packages/extension-ado`? Also, would we publish some form of this package to `npm` (at least for docs)?

**Comment by molant** [comment]
Yes, that's what I was thinking so we can get the information on the website.


________________________________
